src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -95,6 +104,21 @@ public static void setupTest() throws Exception {
             createTableManyParams();
             createProcedureManyParams();
             createGetObjectOffsetDateTimeProcedure(stmt);
+            createProcedureZeroParams();
+            createOutOfOrderSproc();
+            createByParamNameSproc();
+            createUserDefinedFunction();
+        }
+    }
+
+    @Test
+    public void testCallableStatementClosedConnection() throws Exception {
```

Comment:
No exceptions can be thrown.
```suggestion
    public void testCallableStatementClosedConnection() {
```